packages/ui-app/src/Tooltip.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2019 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTooltip from 'react-tooltip';
+
+import { BareProps } from './types';
+
+const rootElement = document.getElementById('tooltips');
+
+type Props = BareProps & {
+  trigger?: string,
+  delayShow?: number,
+  place?: 'bottom' | 'top' | 'right' | 'left',
+  effect?: 'solid' | 'float'
+
+};
+
+export default class Tooltip extends React.PureComponent<Props> {
+
+  static defaultProps = {
+    trigger: 'default-trigger',
+    delayShow: 250,
+    effect: 'solid',
+    place: 'bottom',
+    className: 'ui--Tooltip'
+  };
+
+  tooltipContainer: HTMLElement;
+  constructor (props: Props) {
```

Comment:
(nitpick) Prefer a blank line just before

Reply:
Thanks for the tips I'm trying to improve my code formatting skills :)

Reply:
Making the "trigger" prop mandatory will reflect better the default implementation of the component, so I just updated it in the same commit, hope you don't mind !

Thanks!